notebooks/image_models/solutions/diffusion_model.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        # diffusion times -&gt; angles
</pre><p>Can you expand on this comment? Are you saying you're converting dissusion_times to angles?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>